lang/cs/Org.Apache.REEF.Demo/Task/IDataSetManager.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.Demo.Task
+{
+    public interface IDataSetManager
```

Comment:
This class is for the task-side and not for the driver-side.
From the driver-side, there wouldn't be much usage for fetching specific blocks or partitions; rather, the user would just want to fetch all data, e.g. at the end of the ML pipeline. That's why there is `Collect` on `IDataSet`.